content/cn/docs/clients/restful-api/propertykey.md

```
@@ -6,25 +6,25 @@ weight: 2
-- id：属性类型id值
+- id：属性类型 id 值
 - properties：属性的属性，对于属性而言，此项为空
-- user_data：设置属性类型的通用信息，比如可设置age属性的取值范围，最小为0，最大为100；目前此项不做任何校验，只为后期拓展提供预留入口
+- user_data：设置属性类型的通用信息，比如可设置 age 属性的取值范围，最小为 0，最大为 100；目前此项不做任何校验，只为后期拓展提供预留入口
 
 
 #### 1.2.1 创建一个 PropertyKey
 
 ##### Method & Url
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)